adapters/oidc/js/src/main/resources/keycloak.d.ts

```diff
@@ -49,9 +49,22 @@ declare namespace Keycloak {
 		 * Client identifier, example: 'myapp'
 		 */
 		clientId: string;
+
+		/**
+		 * Scope to use for the 'scope' login url parameter.
+		 * The scope 'openid' will be added to the scope if it is missing or undefined.
+		 */
+		scope?: string;

```

Comment:
Do we need the scope parameter in both `KeycloakConfig` and `KeycloakInitOptions`? `KeycloakConfig` is an equivalent of `AdapterConfig` class for Java adapters and it contains no scope parameter, so I would prefer to remove it, WDYT?

Reply:
I didn't think of that - yes, you are right. I'll remove the prop again from Keycloak config.